src/main/java/com/programmers/ticketparis/exception/ExceptionRule.java

```diff
@@ -14,9 +14,12 @@ public enum ExceptionRule {
     NOT_EXIST_RESERVATION(HttpStatus.NOT_FOUND, "요청한 예매 ID가 존재하지 않음"),
     NOT_EXIST_RESERVATION_STATUS(HttpStatus.NOT_FOUND, "존재하지 않는 예약 상태"),
 
-    NOT_EXIST_CUSTOMER(HttpStatus.NOT_FOUND, "요청한 회원(구매자) ID가 존재하지 않음"),
+    NOT_EXIST_CUSTOMER(HttpStatus.NOT_FOUND, "요청한 구매자 ID가 존재하지 않음"),
```

Comment:
`p1`

팀 컨벤션 페이지에 나와있듯 Rule에 대한 네이밍은 `도메인` + `NOT_EXIST`의 방식으로 해주시면 좋을 것 같습니다! 👍👍
 ```suggestion
    CUSTOMER_NOT_EXIST(HttpStatus.NOT_FOUND, "요청한 구매자 ID가 존재하지 않음"),
```

Reply:
앗 수정했습니다ㅎㅎ. 감사합니다 
25c521296736e7ec5c26cc732e40c161352f9efc